xml/System/Convert.xml

```diff
@@ -18590,12 +18595,14 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="bytes">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">The string representation that is encoded with base-64 digits.</param>
+        <param name="bytes">When this method returns <see langword="true" />, the converted 8-bit unsigned integers.</param>
+        <param name="bytesWritten">When this method returns <see langword="true" />, the number of bytes that were written in <paramref name="bytes" />. 0 when this method returns <see langword="false" />.</param>
```

Comment:
I think this can be simplified 

```suggestion
        <param name="bytesWritten">When this method returns, the number of bytes that were written in <paramref name="bytes" />.</param>
```